samtranslator/metrics/metrics.py

```diff
@@ -3,6 +3,7 @@
 """
 import logging
 from datetime import datetime
+from aws_embedded_metrics import metric_scope
```

Comment:
Were does this module come from? I don't think this is anything in SAM and therefore will cause an import error and crash.

Reply:
@jfuss Thanks for the comments. We already have another mechanism to collect the IAM metrics, so close this pr.